win32/src/winapi/stack_args.rs

```diff
@@ -209,3 +209,8 @@ impl ToX86 for i32 {
         *self as u32
     }
 }
+impl ToX86 for () {
```

Comment:
Man, why didn't I do this already!  Good idea.

Reply:
BTW I merged this here https://github.com/evmar/retrowin32/commit/2a347bbab84cf98ea6fbc7dcdfb06126e396f783